index.html

```diff
@@ -1086,8 +1283,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_SECTION</code> and object attribute
 											<code>xml-roles:doc-qna</code>. </p></td>
-								<td>AXRole: <code>AXGroup</code><br /> AXSubrole: <code>AXApplicationGroup</code><br />
-									AXRoleDescription: <code>'group'</code>
+								<td>
+									<ul>
+										<li>AXRole: <code>AXGroup</code></li>
+										<li>AXSubrole: <code>AXApplicationGroup</code></li>
+										<li>AXRoleDescription: <code>'group'</code></li>
+										<li>AXCustomContent:<code>{ label: "type", value: "Q&amp;A" }</code></li>
```

Comment:
TBD… expand abbreviation?

Reply:
That sounds fine to me. It's truncated to "qna" for typing purposes but doesn't need to be identical for pronunciation.

Reply:
Confirming as-is. "Questions & Answers" is too verbose.

Reply:
+1